email_mailbox/src/components/PanelWrapper.js

```diff
@@ -269,6 +269,10 @@ class PanelWrapper extends Component {
       Event.UPDATE_THREAD_EMAILS,
       this.updateThreadEmailsListenerCallback
     );
+    addEvent(
```

Comment:
This can be an overheat if I receive too many `Is Trusted Peer Event`. I recommend to update all contacts as a batch after `getEvents` is done, as it is with modified labels

Reply:
Even i understand the problem, it will require some work because events not give me the id, only the emails, and i need to map each contactId. Saying that i no consider that in this case we will receive a lot of isTrusted peer events.